indigo_api/serializers.py

```diff
@@ -572,6 +578,22 @@ def get_amendments_url(self, work):
             return None
         return reverse('work-amendments-list', request=self.context['request'], kwargs={'work_id': work.pk})
 
+    def get_taxonomies(self, instance):
+        taxonomies = []
+        topics = instance.taxonomies.get_queryset().prefetch_related('taxonomy_vocabulary')
```

Comment:
It's crucial that these are ordered by the same thing that `groupby` will group them by. So you'll need `.order_by('taxonomy_vocabulary_id')` at the end.